### PR TITLE
Add tests on products page

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add tests on products page :  preview, duplicate, bulk actions
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/03_catalog/01_products/*" URL_FO=ShopUrl/  npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22345)

### CODE REVIEW

File: tests/UI/pages/BO/catalog/products/add.js [hunk 1]
```diff
@@ -464,6 +467,15 @@ class AddProduct extends BOBasePage {
       await this.addProductToPack(page, keys[i], pack[keys[i]]);
     }
   }
+
+  /**
+   * Get product name frrom inout
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/products/add.js]:
```suggestion
   * Get product name from input
```

File: tests/UI/pages/BO/catalog/products/index.js [hunk 2]
```diff
@@ -429,6 +478,27 @@ class Product extends BOBasePage {
...
+   * @return {Promise<string>}
+   */
+  async bulkSetStatus(page, status) {
+    await Promise.all([
+      this.waitForVisibleSelector(page, this.productBulkMenuButton),
+      page.click(this.selectAllBulkCheckboxLabel),
+    ]);
+
+    await Promise.all([
+      this.waitForVisibleSelector(page, `${this.productBulkMenuButton}[aria-expanded='true']`),
```
Comment (SimonGrn) [inline tests/UI/pages/BO/catalog/products/index.js]: Can't we make this selector an attribute ?

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/pages/BO/catalog/products/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/products/index.js]: done

File: tests/UI/pages/BO/catalog/products/add.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/products/add.js]: done

**Review by SimonGrn — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
